src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,32 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
+        Injector injector = Injector.getInstance("mate.jdbc");
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setCountry("Ukraine");
+        manufacturer.setName("Stepan");
+        manufacturerDao.create(manufacturer);
+
+        manufacturerDao.get(2L);
```

Comment:
```suggestion
        manufacturerDao.get(manufacturerOne.getId());
```
lets replace magic numbers with getId()